tests/test_bytes.rs

```diff
@@ -291,6 +291,7 @@ fn split_to_uninitialized() {
 }
 
 #[test]
+#[cfg_attr(not(panic = "unwind"), ignore)]
```

Comment:
```suggestion
#[cfg_attr(panic = "abort", ignore)]
```

Reply:
There could be additional panic strategies in the future.